TombLib/TombLib/Catalogs/TEN Node Catalogs/_System.lua

```diff
@@ -92,4 +92,37 @@ LevelFuncs.Engine.Node.GetRoomFlag = function(value)
 	if (value == 6) then return Objects.RoomFlagID.NOLENSFLARE end
 
 	return 0;
-end
\ No newline at end of file
+
+end
+
+-- Helper function for converting UI blend mode enum to real enum.
+
+LevelFuncs.Engine.Node.GetBlendMode = function(index)
```

Comment:
As stated above, I would remove `TEN.Effects.BlendID.NOZTEST` and `TEN.Effects.BlendID.WIREFRAME` from this enumeration. Also, brackets are missing again! I wonder how it even works this way... this is clearly a syntax error, according to a Lua documentation.